### PR TITLE
[Feature] Set up CI workflow + add linting and prettier scripts

### PR DESCRIPTION
**Pull request by j-bunk**
- Added a CI workflow that builds code + runs prettier and lint checks
-  Added linting and prettier scripts to `package.json`
- Added PR template (when you add a pull request template to your repository, you will automatically see the template's contents in the pull request body)

### CODE REVIEW

File: .github/workflows/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+## Notion ticket link
+<!-- Please replace with your ticket's URL -->
+[Ticket Name](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)
```
Comment (OustanDing) [inline .github/workflows/pull_request_template.md:3]:
```suggestion
[Ticket_Name](https://www.notion.so/uwblueprintexecs/ac7b1f636a214a0cbf45f6b76f899b54?v=5414ee5fc5974f149921e192d6eb29b8)
```

Replace with our task board

File: .github/workflows/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+<!-- What should the reviewer do to verify your changes? Describe expected results and include screenshots when appropriate -->
+## Steps to test
+1.
+
+
+<!-- Draw attention to the substantial parts of your PR or anything you'd like a second opinion on
+For Frontend changes be sure to specify which routes/pages reviewers should check out!
+ -->
+## What should reviewers focus on?
+*
```
Comment (OustanDing) [inline .github/workflows/pull_request_template.md:20]: Don't need this

File: .github/workflows/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+For Frontend changes be sure to specify which routes/pages reviewers should check out!
+ -->
+## What should reviewers focus on?
+* 
+
+
+## Checklist
+- [ ] My PR name is descriptive, is in imperative tense and starts with one of the following: `[Feature]`,`[Improvement]` or `[Fix]`,
+- [ ] I have run the appropriate linter(s)
+- [ ] I have requested a review from rcd
```
Comment (OustanDing) [inline .github/workflows/pull_request_template.md:26]:
```suggestion
- [ ] I have requested a review from the RCD team on GitHub, or specific people who are associated with this ticket
```

File: .github/workflows/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+
+<!-- Give a quick summary of the implementation details, provide design justifications if necessary -->
+## Implementation description
+* 
+
+
+<!-- What should the reviewer do to verify your changes? Describe expected results and include screenshots when appropriate -->
+## Steps to test
+1.
```
Comment (OustanDing) [inline .github/workflows/pull_request_template.md:13]: Could you please replace this section with a "Notes" section? Just generic bullet list of things that reviewers should keep in mind, things that will be fixed in the future, things that were left out, etc

File: .github/workflows/pull_request_template.md (hunk 1 above)
Comment (j-bunk) [reply .github/workflows/pull_request_template.md]: Addressed!

File: .github/workflows/pull_request_template.md (hunk 2 above)
Comment (j-bunk) [reply .github/workflows/pull_request_template.md]: Addressed!

File: .github/workflows/pull_request_template.md (hunk 3 above)
Comment (j-bunk) [reply .github/workflows/pull_request_template.md]: Addressed!

File: .github/workflows/pull_request_template.md (hunk 4 above)
Comment (j-bunk) [reply .github/workflows/pull_request_template.md]: Addressed!

**Review by OustanDing — APPROVED** [review]
LGTM!

**Pull request merged**